doc/source/whatsnew/v0.25.1.rst

```diff
@@ -12,7 +12,7 @@ Enhancements
 Other enhancements
 ^^^^^^^^^^^^^^^^^^
 
--
+- Changed :meth:`MultiIndex.from_product` to infer level names from inputs if possible
```

Comment:
Can you add the issue number to the end of this line: <code>(:issue:\`27292\`)</code>

Reply:
Can you also move this to the 1.0 whatsnew?  We generally try to keep the minor releases to just bug fixes and regressions.

Reply:
Did I have it in the right place apart from being in the right file?

Reply:
Yes, the "Other enhancements" is the correct place to put this.

Reply:
Added to other enhancements in the 1.0 whatsnew